theme.inc

```diff
@@ -76,3 +77,32 @@ function _gent_base_get_entity_reference_file_data(&$variables, $field) {
 
   return $data;
 }
+
+/**
+ * Helper function to get the file data of a referenced File entity.
+ *
+ * @param mixed $item
+ *   The item to get the referenced entities from.
+ *
+ * @return array|null
+ *   The terms that are referenced.
+ *
+ * @throws \Drupal\Core\TypedData\Exception\MissingDataException
+ */
+function _gent_base_get_related_terrains($item) {
+  // Get the related terrain node.
+  $node = Node::load($item->id());
+  if (!$node) {
+    return;
+  }
+
+  $terms = [];
+
+  // Get the park offer field value.
+  $parkItems = $node->get('field_park_offer')->referencedEntities();
```

Comment:
referencedEntities() does not make a distinction between any of the references.
If later on another entity reference field is added, it will also be listed.

See https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21Entity.php/function/Entity%3A%3AreferencedEntities/8.6.x